### PR TITLE
🌐 [i18n-KO] Translated `model_doc/cohere.md` to Korean

### PR DESCRIPTION
**Pull request by fabxoe**
# What does this PR do?

Translated the `model_doc/cohere.md` file of the documentation to Korean.
Thank you in advance for your review. 

Part of https://github.com/huggingface/transformers/issues/20179

## Before reviewing
- [x] Check for missing / redundant translations (번역 누락/중복 검사)
- [x] Grammar Check (맞춤법 검사)
- [x] Review or Add new terms to glossary (용어 확인 및 추가)
- [x] Check Inline TOC (e.g. `[[lowercased-header]]`)
- [ ] Check live-preview for gotchas (live-preview로 정상작동 확인)

## Who can review? (Initial)

@chhaewxn, @ahnjj, @jun048098, @fabxoe, @nuatmochoi, @heuristicwave

## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [x] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review? (Final)

@stevhliu May you please review this PR?

### CODE REVIEW

File: docs/source/ko/model_doc/cohere.md [hunk 1]
```diff
@@ -0,0 +1,139 @@
+# Cohere[[cohere]]
+
+## 오버뷰[[overview]]
```
Comment (jun048098) [inline docs/source/ko/model_doc/cohere.md:3]:
```suggestion
## 개요[[overview]]
```

**Review by jun048098 — COMMENTED** [review]
fix: docs suggestion

File: docs/source/ko/model_doc/cohere.md [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+    max_new_tokens=100, 
+    do_sample=True, 
+    temperature=0.3,
+    )
+
+gen_text = tokenizer.decode(gen_tokens[0])
+print(gen_text)
+```
+
+bitsnbytes 라이브러리를 이용해서 4bit 양자화된 모델 로딩
```
Comment (nuatmochoi) [inline docs/source/ko/model_doc/cohere.md:92]:
```suggestion
bitsandbytes 라이브러리를 이용해서 4bit 양자화된 모델 로딩
```

File: docs/source/ko/model_doc/cohere.md [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+- RAG 및 도구 사용에 대한 강력한 정확성
+- 낮은 지연 시간과 높은 처리량
+- 더 긴 128k 컨텍스트와 낮은 가격
+- 10개의 주요 언어에 걸친 강력한 기능
+- 연구 및 평가를 위해 HuggingFace에서 사용 가능한 모델 가중치
+
+모델 체크포인트는 [이곳](https://huggingface.co/CohereForAI/c4ai-command-r-v01)에서 확인하세요.
+이 모델은 [Saurabh Dash](https://huggingface.co/saurabhdash)과 [Ahmet Üstün](https://huggingface.co/ahmetustun)에 의해 기여 되었습니다. Hugging Face에서 이 코드의 구현은 [GPT-NeoX](https://github.com/EleutherAI/gpt-neox)에 기반하였습니다.
+
+## 사용팁[[usage-tips]]
```
Comment (fabxoe) [inline docs/source/ko/model_doc/cohere.md:21]:
```suggestion
## 사용 팁[[usage-tips]]
```

**Review by stevhliu — APPROVED** [review]
Thanks! 🤗

**Pull request merged**